corehq/apps/hqwebapp/templates/hqwebapp/downgrade_modal.html

```diff
@@ -0,0 +1,38 @@
+{% load hq_shared_tags %}
+{% load i18n %}
+
+<div class="modal" id="downgrade-modal" data-slug="downgrade_pending" data-domain="{{ domain }}">
+    <div class="modal-dialog">
+        <div class="modal-content">
+            <div class="modal-header {% if days_until_downgrade <= 30 %}bg-danger text-danger{% endif %}">
+                <button type="button" class="close" data-dismiss="modal">
+                    <span aria-hidden="true">×</span>
+                    <span class="sr-only">Close</span>
+                </button>
+                <h4 class="modal-title">
+                    Your invoice for {{ invoice_month }} is past due!
```

Comment:
`trans` (throughout this file)

Reply:
Updated: fc4734996cea51cf88168df0d1917c55779e77cf